src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1481,9 +1500,21 @@ void Convolution::prepareParams() {
 
     if (execPtr) {
         primArgs[DNNL_ARG_SRC] = srcMemPtr->GetPrimitive();
-        primArgs[DNNL_ARG_WEIGHTS] = wghMemPtr->GetPrimitive();
         primArgs[DNNL_ARG_DST] = dstMemPtr->GetPrimitive();
 
+        if (key.constWeight) {
+            // const weight preparation/reordering needs to be done once at next execution
+            // when the input weight data is guaranteed to be ready (considering possible const-folding
+            // subgraphs inserted between constant weight node and conv)
+            auto it = primArgs.find(DNNL_ARG_WEIGHTS);
+            if (it == primArgs.end() || prevExecPtr->getWeightDesc() != execPtr->getWeightDesc()) {
```

Comment:
After the rebase to the new version of the DnnlExecutor, `prevExecPtr->getWeightDesc() != execPtr->getWeightDesc()` will have the semantics of comparison of two pointers, so please change this check to `!execPtr->getWeightDesc()->isCompatible(*(prevExecPtr->getWeightDesc()))`

Reply:
OK, will change!